controller:
  ros__parameters:
    alt_hz: 20.0
    alt_toz: 5.0
    tau: 50.0
    c_kp: 2.0
    c_kd: 0.0
    c_ki: 0.0025
    r_kp: 0.55
    r_kd: 0.20
    r_ki: 0.0
    p_kp: -4.25
    p_kd: -0.6
    p_ki: 0.0
    p_ff: 0.0
    a_t_kp: 0.1
    a_t_kd: 0.0
    a_t_ki: 0.02
    a_kp: 0.015
    a_kd: 0.0
    a_ki: 0.001
    l_kp: .5
    l_kd: 0.0
    l_ki: 0.005
    e_kp: .5
    e_kd: 0.0
    e_ki: .01
    trim_e: 0.075
    trim_a: 0.0
    trim_r: 0.0
    trim_t: 0.5
    max_e: 0.61
    max_a: 0.8
    max_r: 0.523
    max_t: 1.0
    pwm_rad_e: 1.0
    pwm_rad_a: 1.0
    pwm_rad_r: 1.0
    max_takeoff_throttle: 1.0
    takeoff_airspeed: 25.0
    mass: 4.5
    gravity: 9.8
    max_roll: 45.0
    controller_output_frequency: 100.0
path_manager:
  ros__parameters:
    R_min: 50.0
    orbit_last: False
    default_altitude: 50.0
    default_airspeed: 15.0
    current_path_pub_frequency: 100.0
path_follower:
  ros__parameters:
    controller_commands_pub_frequency: 10.0
    chi_infty: 1.2
    k_orbit: 0.5
    k_path: 0.0175
    gravity: 9.81
estimator:
  ros__parameters:
    rho: 1.225
    gravity: 9.8
    sigma_n_gps: 0.01
    sigma_e_gps: 0.01
    sigma_Vg_gps: 0.005
    sigma_course_gps: 0.00025
    sigma_accel: 0.024525
    lpf_a: 50.0
    lpf_a1: 8.0
    gps_n_lim: 10000.
    gps_e_lim: 10000.
    frequency: 100
    baro_calibration_val: 85271.15625
    init_lat: 40.246377290528969
    init_lon: -111.6474166337984
    init_alt: 1387.218994140625